Expecto.sln

```diff
@@ -1,76 +1,83 @@
-		Gemfile = Gemfile
-		globals.json = globals.json
-		paket.dependencies = paket.dependencies
-		Rakefile = Rakefile
-		README.md = README.md
-	EndProjectSection
-EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{5B9D0E5F-378A-4310-B9F3-4ECF67AA5A23}"
-EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "sample", "sample", "{191A316D-EA59-4BA9-9A70-D2AC46ECC6ED}"
-EndProject
-Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Expecto.Tests", "Expecto.Tests\Expecto.Tests.fsproj", "{4182E778-F6C2-4BA4-BFC7-19DB54890E18}"
-EndProject
-Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Expecto.Sample", "Expecto.Sample\Expecto.Sample.fsproj", "{9996E3A7-F97F-4A6A-AD5F-C6C70858E220}"
-EndProject
-Global
-	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|Any CPU = Debug|Any CPU
-		Release|Any CPU = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{C0D55728-10A9-4A7A-9DF9-D2F21F663AC2}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{C0D55728-10A9-4A7A-9DF9-D2F21F663AC2}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{C0D55728-10A9-4A7A-9DF9-D2F21F663AC2}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{C0D55728-10A9-4A7A-9DF9-D2F21F663AC2}.Release|Any CPU.Build.0 = Release|Any CPU
-		{0ACABEAA-29BD-4F9F-89DB-0D03B5AFC952}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{0ACABEAA-29BD-4F9F-89DB-0D03B5AFC952}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{0ACABEAA-29BD-4F9F-89DB-0D03B5AFC952}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{0ACABEAA-29BD-4F9F-89DB-0D03B5AFC952}.Release|Any CPU.Build.0 = Release|Any CPU
-		{58507646-335A-460E-B283-77A2BAF0D3A8}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{58507646-335A-460E-B283-77A2BAF0D3A8}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{58507646-335A-460E-B283-77A2BAF0D3A8}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{58507646-335A-460E-B283-77A2BAF0D3A8}.Release|Any CPU.Build.0 = Release|Any CPU
-		{4182E778-F6C2-4BA4-BFC7-19DB54890E18}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{4182E778-F6C2-4BA4-BFC7-19DB54890E18}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{4182E778-F6C2-4BA4-BFC7-19DB54890E18}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{4182E778-F6C2-4BA4-BFC7-19DB54890E18}.Release|Any CPU.Build.0 = Release|Any CPU
-		{9996E3A7-F97F-4A6A-AD5F-C6C70858E220}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{9996E3A7-F97F-4A6A-AD5F-C6C70858E220}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{9996E3A7-F97F-4A6A-AD5F-C6C70858E220}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{9996E3A7-F97F-4A6A-AD5F-C6C70858E220}.Release|Any CPU.Build.0 = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-	GlobalSection(NestedProjects) = preSolution
-		{4182E778-F6C2-4BA4-BFC7-19DB54890E18} = {5B9D0E5F-378A-4310-B9F3-4ECF67AA5A23}
-		{9996E3A7-F97F-4A6A-AD5F-C6C70858E220} = {191A316D-EA59-4BA9-9A70-D2AC46ECC6ED}
-	EndGlobalSection
-EndGlobal
+﻿

```

Comment:
1. Didn't understand the comment about "not executable"
2. Visual Studio switched the endings, didn't want to edit by hand.